schedule/lvm_thin_provisioning.yml

```diff
@@ -0,0 +1,77 @@
+name:           lvm_thin_provisioning
+description:    >
+    Complete OS deployment with unencrypted LVM drive management.
+    Test creates 2 LVM and BIOS boot partitions.
+    Thin pool and thin lv resides on the second LVM partition,
+    where /home (XFS) is being mounted. Maintainer: mloviska, poo#39023
+conditional_schedule:
+    bootloader:
+        ARCH:
+            x86_64:
+                - installation/isosize
+                - installation/bootloader
+            s390x:
+                - installation/bootloader_zkvm
+            aarch64:
+                - installation/isosize
+                - installation/bootloader_uefi
+    addons_repos:
+        DISTRI:
+            opensuse:
+                - installation/online_repos
+                - installation/installation_mode
+                - installation/logpackages
+            sle:
+                - installation/scc_registration
+                - installation/addon_products_sle
+    hostname_inst:
+        DISTRI:
+            sle:
+                - installation/hostname_inst
+    user_settings_root:
+        DISTRI:
+            sle:
```

Comment:
why only for SLE?

Reply:
Same here, don't ask me, dig git history. And I'm with you, we should resolve that, but, please, don't pollute PoC with questions which are out of scope here.